students/johnpharmd/lesson01/break_me.py

```diff
@@ -0,0 +1,23 @@
+def funct1():
+    a = 8
+    print(ab)
+
+
+def funct2():
+    b = '5'
+    print(5 + b)
+
+
+def funct3():
+	print('hello, world!)
+
+
+def funct4():
+    string = 'a, b, c, d'
```

Comment:
not important for this exercise but something to keep in mind

`string` is a built in module and need to be careful about overwriting built-in modules or reserved keywords. github is nice enough to syntax highlight them for you : ) (its in orange color)

here is example of what happens:
```
>>> import string
>>> string.digits
'0123456789'
>>> string = 'my string!'
>>> string.digits
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'str' object has no attribute 'digits'
>>>```